packages/builder/src/polywrap_client_config.rs

```diff
@@ -10,10 +10,10 @@ use polywrap_core::{
 };
 use polywrap_resolvers::static_resolver::{StaticResolver, StaticResolverLike};
 
-use crate::{PolywrapBaseResolver, PolywrapBaseResolverOptions, PolywrapClientConfigBuilder};
+use crate::{PolywrapBaseResolver, PolywrapBaseResolverOptions, ClientConfigBuilder};
 
 #[derive(Default, Clone)]
-pub struct PolywrapClientConfig {
+pub struct ClientConfig {
```

Comment:
i feel that this should be something like `ClientBuilderConfig` and the `CoreClientConfig` be `ClientConfig` - I feel that the `Core` prefix is non intuitive. Wdyt? Also cc to @namesty and @nerfZael to see what they think

Reply:
I think we should make the most user-facing type the easiest to understand and read. So, that would mean that the config used by the builder is the `ClientConfig`, and the "fully built config" that's used by the client would be the `CoreClientConfig`.

Reply:
> I think we should make the most user-facing type the easiest to understand and read. So, that would mean that the config used by the builder is the `ClientConfig`, and the "fully built config" that's used by the client would be the `CoreClientConfig`.

yah in my opinion that's not the easiest to understand and read but rather confusing. if I see this for the first time I would ask to myself "what's the difference between a client and a core client" and why do they have different attributes. from my perspective, this can be really simplified to say that one is the builder attributes and the other is the client attributes (but again, this is my opinion and I can be completely wrong)

Reply:
anyways, I will merge this because it is a nit, and there's no need to merge this